src/MongoDB.Driver.Core/ServerErrorCode.cs

```diff
@@ -44,6 +44,7 @@ internal enum ServerErrorCode
         UnknownReplWriteConcern = 79,
         UnsatisfiableWriteConcern = 100,
         WriteConcernFailed = 64,
-        MaxTimeMSExpired = 50
+        MaxTimeMSExpired = 50,
+        NamespaceNotFound = 26
```

Comment:
Looks like we were trying to keep these in alphabetical order.

Reply:
done